sdk/azidentity/chained_token_credential.go

```diff
@@ -18,42 +19,45 @@ type ChainedTokenCredential struct {
 
 // NewChainedTokenCredential creates an instance of ChainedTokenCredential with the specified TokenCredential sources.
 func NewChainedTokenCredential(sources ...azcore.TokenCredential) (*ChainedTokenCredential, error) {
```

Comment:
This is no-fail now so the `error` return value can be removed.

Reply:
Very right, thanks for catching that! I'm removing it.